props/saber_fett263_buttons.h

```diff
@@ -2852,8 +2870,16 @@ SaberFett263Buttons() : PropBase() {}
         itoa(style_num_, style_arg, 10);
         current_preset_.SetStyle(blade_num_, style_parser.SetArgument(current_preset_.GetStyle(blade_num_), 1, style_arg));
         current_preset_.Save();
-        sound_library_.SayNumber(style_num_, SAY_WHOLE);
         UpdateStyle(current_preset_.preset_num);
+        sound_library_.SayStyle();
+        sound_library_.SayNumber(style_num_, SAY_WHOLE);
```

Comment:
This part looks very similar to line 2119 above, maybe break it out into a helper function and save a few bytes?

Reply:
broke out to SayStyleNumber()